aries-site/src/examples/components/fileinput/FileInputSimpleExample.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Box, FileInput } from 'grommet';
+
+export const FileInputSimpleExample = () => (
+  <Box align="center" pad="large">
```

Comment:
Can we set a width of `medium` on this box so the file input is a little wider by default? Right now it's quite narrow:
<img width="1414" alt="Screen Shot 2021-05-03 at 11 57 07 AM" src="https://user-images.githubusercontent.com/12522275/116920133-21fe6200-ac07-11eb-9d01-808d952f8207.png">

Reply:
I think we should still set a fixed width so it doesn't grow and shrink as files are dragged onto it